biz.aQute.repository/src/aQute/maven/provider/POM.java

```diff
@@ -58,6 +58,9 @@
 	private Map<Program, Dependency>	dependencies			= new LinkedHashMap<>();
 	private Map<Program, Dependency>	dependencyManagement	= new LinkedHashMap<>();
 	private XPath						xp;
+	private String						JAR_PACKAGING[]			= {
```

Comment:
We normally use `String[] var;` not `String var[];`.

Reply:
I've changed it because I do not want to fight over it but if you want to enforce these things you should start a style guide where we can agree on these things because this feels very arbitrary.